### DISCUSSION TITLE
Welcome to VisualTeensy discussions

### DISCUSSION THREAD

**Discussion by luni64** [luni64/VisualTeensy#34] [General]
## 👋 Welcome!
 You can use Discussions as a place to connect with other members of the community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.

**Comment by makeller1** [comment]
Hi Luni, I enabled 8.3 file naming to fix the error I had and learned something you may want to pass along in your quick start guide: after enabling 8.3 naming, only new folders receive a short name. So the contents must be copied to a new folder after enabling. Thanks again!
